worker/uploader.go

```diff
@@ -158,23 +169,28 @@ func handleSectorUpload(uploadErr error, uploadDuration, totalDuration time.Dura
 		return true, false, float64(ms), float64(rhpv2.SectorSize / ms)
 	}
 
+	// upload failed because we weren't able to create a payment, in this case
+	// we want to punish the host but only to ensure we stop using it, meaning
+	// we don't increment consecutive failures
+	if utils.IsErr(uploadErr, errFailedToCreatePayment) {
```

Comment:
This seems like quite the harsh punishment for something that is hopefully fixed by the autopilot performing a refresh. Does the worker ever recover from a 1 hour punishment?

Reply:
The `autopilot` 's default heartbeat is quite high, so worst case we're stuck 30 minutes, and the upload estimate stats have a decay where the half time is set at 10 minutes. I'd have to do the math but I'm quite confident this should be ok.